packages/graphql-transformer-common/src/PredictionsResourceIDs.ts

```diff
@@ -5,7 +5,7 @@ export class PredictionsResourceIDs {
   static lambdaName = 'predictionsLambda';
   static lambdaID = 'predictionsLambdaFunction';
   static lambdaHandlerName = 'predictionsLambda.handler';
-  static lambdaRuntime = 'nodejs14.x';
+  static lambdaRuntime = 'nodejs18.x';
```

Comment:
one thing to double check - does this or conflict handler rely on AWS SDK. If yes then code should be adapted to use V3 instead of V2.

I skimmed both functions and they don't seem to rely on SDK, so we should be fine. But please double check that.